dap-etl/extraction/src/main/scala/org/broadinstitute/monster/dap/cslb/CslbExtractionPipeline.scala

```diff
@@ -30,7 +30,9 @@ object CslbExtractionPipeline extends ScioApp[Args] {
         .getOrElse(List())
 
   val subdir = "cslb"
-  val arm = List("annual_2020_arm_1")
+
+  val arm =
+    List("annual_2020_arm_1", "annual_2021_arm_1")
```

Comment:
What do you think about parameterizing these as cli args so we don't have to PR these sorts of changes going forward? I foresee a scenario where we'll be asked to pull the 2020_arm1 again for whatever reason and a full PR cycle would be cumbersome.